iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepKeyword.java

```diff
@@ -23,6 +23,8 @@
 
 public enum BicepKeyword implements GrammarRuleKey {
 
+  EXISTING("existing"),
+  RESOURCE("resource"),
```

Comment:
Please check, if the keywords in Bicep are case-sensitive or not.

Reply:
The keywords are case-sensitive (checked for `existing` in deployment, also VS code mark it as error)

Reply:
Okay, thanks. "VS code mark it as error" is not an argument, since they have their own parser implementation. IntelliJ is not able to parse lowercase instructions for Docker which are valid.

Reply:
Deployment fails with `EXISTING` instead of `existing` (this is what I mean "checked for existing in deployment"). 
I was unable to find a confirmation in the documentation.